resources/assets/js/forms.js

```diff
@@ -200,10 +200,15 @@ ready(() => {
 });
 
 ready(() => {
-  $('select').select2({
-    theme: 'bootstrap-5',
-    width: '100%',
-  });
+  document.querySelectorAll('select').forEach(
+    (element) =>
```

Comment:
i would put the `(element) =>` into the previous line

Reply:
Code style in JS/CSS is handled by [prettier](https://prettier.io/).